content/rdi/architecture.md

```diff
@@ -9,59 +9,89 @@ headerRange: "[2]"
 aliases: 
 ---
 
-Redis Data Integration is a product for data ingestion from existing systems of record (mainly databases) into Redis Enterprise.
-You can think of Redis Data Integration as a streaming ELT process:
+Redis Data Integration (RDI) is a product that helps Redis Enterprise users ingest and export data in near real time. RDI provides the following:
```

Comment:
Consider having the "RDI provides the following" in a new line (probably with another empty line) to have it in another section.

Reply:
done